### PR TITLE
Fixing codacy issues

### PR DESCRIPTION
**Pull request by Log234**
Fixed a number of code issues reported by Codacy. Most of the issues currently in the code should be fixed with this merge.

### CODE REVIEW

File: Lavinia-api/Data/CustomValidation.cs [hunk 1]
```diff
@@ -15,22 +15,43 @@ public static class CustomValidation
...
 
             if (country.InternationalName == null)
+            {
                 throw new ArgumentException("Country.InternationalName cannot be null.");
+            }
+
             if (country.InternationalName.Length < 3)
+            {
                 throw new ArgumentException("Country.InternationalName cannot be shorter than 3 characters.");
-            if (country.CountryCode == null) throw new ArgumentException("Country.ShortName cannot be null.");
```
Comment (simendag) [inline Lavinia-api/Data/CustomValidation.cs]: This change does not seem to match the description of the commit.

File: Lavinia-api/Data/CustomValidation.cs [hunk 2]
```diff
@@ -42,18 +63,40 @@ public static void ValidateCountry(Country country, HashSet<int> checkedModels)
...
+            {
+                throw new ArgumentException("County cannot be null.");
+            }
+
+            if (county.Name == null)
+            {
+                throw new ArgumentException("County.Name cannot be null.");
+            }
 
-            if (county.Name == null) throw new ArgumentException("County.Name cannot be null.");
```
Comment (simendag) [inline Lavinia-api/Data/CustomValidation.cs]: These changes seem completely unrelated to the commit message.

File: Lavinia-api/Data/CustomValidation.cs [hunk 3]
```diff
@@ -64,12 +107,25 @@ private static void ValidateCounty(County county, HashSet<int> checkedModels)
...
+
+            if (countyData.Year == -1)
+            {
+                throw new ArgumentException("CountyData.Year cannot be the default value.");
+            }
 
-            if (countyData.Year == -1) throw new ArgumentException("CountyData.Year cannot be the default value.");
             if (countyData.Population == -1)
+            {
                 throw new ArgumentException("CountyData.Population cannot be the default value.");
```
Comment (simendag) [inline Lavinia-api/Data/CustomValidation.cs]: Changing text in an exception is a refactor, as well as unrelated to the commit's message.

File: Lavinia-api/Program.cs [hunk 4]
```diff
@@ -18,7 +18,7 @@ public static void Main(string[] args)
                 try
                 {
                     ElectionContext context = services.GetRequiredService<ElectionContext>();
-                    ILogger logger = services.GetRequiredService <ILogger<Program>>();
```
Comment (simendag) [inline Lavinia-api/Program.cs]: This change seems unrelated to the commit message and should have been placed in its own commit.

File: Lavinia-api/Startup.cs [hunk 5]
```diff
@@ -13,7 +13,8 @@ namespace LaviniaApi
 {
     public class Startup
     {
-        IHostingEnvironment _env;
```
Comment (simendag) [inline Lavinia-api/Startup.cs]: This change is unrelated to the commit message.

File: Lavinia-api/Utilities/CsvFormats.cs [hunk 6]
```diff
@@ -113,8 +113,8 @@ public ResultFormat Parse(string line, FieldParser parser)
     /// </summary>
     public class CountryFormat : ICsvFormat<CountryFormat>
     {
-        public string CountryCode;
```
Comment (simendag) [inline Lavinia-api/Utilities/CsvFormats.cs]: If I recall correctly these properties are set in constructor only -- it is then more appropriate to use a public readonly get and have no setter to ensure no one thinks it is OK to use the private setters in the method.

File: Lavinia-api/Utilities/CsvFormats.cs [hunk 7]
```diff
@@ -169,11 +169,11 @@ public ElectionTypeFormat Parse(string line, FieldParser parser)
     /// </summary>
     public class CountyDataFormat : ICsvFormat<CountyDataFormat>
     {
-        public double Areal;
-        public string County;
-        public int Population;
-        public int Seats;
-        public int Year;
+        public double Areal { get; private set; }
```
Comment (simendag) [inline Lavinia-api/Utilities/CsvFormats.cs]: This isn't really a problem with this commit, rather a problem from before -- in English it is called "Area", not "Areal"

File: Lavinia-api/Tests/CustomValidationTests.cs [hunk 8]
```diff
@@ -6,13 +6,13 @@
 
 namespace Mandater.Tests
 {
-    public class CustomValidationTests
```
Comment (simendag) [inline Lavinia-api/Tests/CustomValidationTests.cs]: These utility classes might be better refactored as extension methods.

File: Lavinia-api/Data/CustomValidation.cs (hunk 1 above)
Comment (Log234) [reply Lavinia-api/Data/CustomValidation.cs]: The git lines are misaligned, if you look further down you can see that the change is simply adding curly braces to the if.

File: Lavinia-api/Data/CustomValidation.cs (hunk 2 above)
Comment (Log234) [reply Lavinia-api/Data/CustomValidation.cs]: The git lines are misaligned, if you look at line 71 in the new version you will find the same if, but with curly braces.

File: Lavinia-api/Data/CustomValidation.cs (hunk 3 above)
Comment (Log234) [reply Lavinia-api/Data/CustomValidation.cs]: The text in the exception on this line wasn't changed.

Comment (simendag) [reply Lavinia-api/Data/CustomValidation.cs]: My bad, then!
